buddyboss/buddyboss-platform
# PROD-794 Subforum topic search issue on mariadb 10

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2539  .

### How to test the changes in this Pull Request:

1. Make sure you have Mariadb 10 as database server
2. Create forums & nested subforums and topics
3. Search for the nested subforum topics in network search
4. See the result showing properly

### Proof Screenshots or Video

https://www.loom.com/share/db930c7813124db9adeffbd8fb499cbe

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
